### ISSUE TITLE
global option syntax differs from help

### ISSUE THREAD

**Issue by mthuurne** [python-poetry/poetry#1502]
- [x] I am on the [latest](https://github.com/sdispater/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.

- openSUSE Linux 15.1
- Poetry 1.0.0b3

## Issue

The help text states that global options go before the command:
```
$ poetry --help
Poetry version 1.0.0b3

USAGE
  poetry [-h] [-q] [-v [<...>]] [-V] [--ansi] [--no-ansi] [-n] <command> [<arg1>] ... [<argN>]

(snip)
```
However, if I try that, Poetry won't execute the command:
```
$ poetry --no-ansi about
USAGE
  poetry help [<command1>] ... [<commandN>]

(snip)
```
But when I put the global option after the command, it works:
```
$ poetry about --no-ansi
Poetry - Package Management for Python

Poetry is a dependency manager tracking local dependencies of your projects and libraries.
See https://github.com/sdispater/poetry for more information.
```

**Comment by finswimmer** [comment]
Hello @mthuurne,

seems to be solved in the meantime.

Thanks a lot for reporting!

fin swimmer

**Comment by mthuurne** [comment]
Which version did you test with? I can still reproduce the problem in Poetry 1.0.3.

**Comment by finswimmer** [comment]
Hello @mthuurne,

it was obviously to late yesterday and I misread the issue. Sorry :)

fin swimmer

**Comment by mthuurne** [comment]
No problem. I'm happy the issue is being looked at.

**Comment by daghemo** [comment]
Same here. That is, this should work:

`poetry --no-ansi new --src myproject`

But I have to write:

`poetry new --src --no-ansi myproject`

**Comment by mkniewallner** [comment]
This is fixed in the to-be-released Poetry 1.2 version.

```console
$ poetry --no-ansi about
Poetry - Package Management for Python

Version: 1.2.0b2
Poetry-Core Version: 1.1.0b2

Poetry is a dependency manager tracking local dependencies of your projects and libraries.
See https://github.com/python-poetry/poetry for more information.

$ poetry --no-ansi new --src myproject
Created package myproject in myproject
```
